app/app/views.py

```diff
@@ -28,6 +28,93 @@ def home(request):
     return render(request, template_name=template, context=context)
 
 
+def get_date_range(start_date, end_date):
+    if (start_date is not None) and (end_date is not None) and (start_date != end_date):
+        date = f"{start_date} - {end_date}"
+    elif (start_date is not None) and (end_date is not None) and (start_date == end_date):
+        date = start_date
+    elif start_date is not None:
+        date = f"Since {start_date}"
+    elif end_date is not None:
+        date = f"Until {end_date}"
+    else:
+        date = None
```

Comment:
This will still need internationalisation, as it contains user facing strings. Unfortunately f-strings are not supported by the extraction tools, so you'll have to go with one of the alternatives. While we're at it, please replace the minus in the first case with an en dash ( – ) which will appear better.